src/components/RegForm/_RegForm.styles.js

```diff
@@ -28,19 +29,68 @@ export const Title = styled.h1`
 export const SubTitle = styled.h2`
 	font-size: 1.15rem;
 	margin-bottom: 1rem;
+
+`
+
+export const Frame = styled.div`
+	position: relative;
+	height: 60px;
+	width: 100%;
+	background: transparent;
+	margin-bottom: 1.5rem;
+`
+
+export const Label = styled.label`
+	position: absolute;
+	z-index: -5;
+	/* top: 1rem; */
+	transform: translateY(100%);
+	margin-left: 0.5rem;
+	color: #8c8c8c;
+
+	${ ({ focus }) => {
+		if (focus) {
+			return css`
+				animation: moveFocus ease-in-out 0.25s;
+				animation-fill-mode: forwards;
+
+				@keyframes moveFocus {
```

Comment:
oh, nice approach my dude! using styled-components `css` w/ `@keyframes`
you can technical modularize the keyframe as another file, import it, then apply it

talk to me if you want to find out how

Reply:
nice, I never thought of it in that way.